validator/keymanager/remote-web3signer/keymanager.go

```diff
@@ -482,8 +488,20 @@ func handleAggregateAttestationAndProof(ctx context.Context, validator *validato
 	return json.Marshal(aggregateAndProofSignRequest)
 }
 
+func handleAggregateAttestationAndProofV2Electra(ctx context.Context, validator *validator.Validate, request *validatorpb.SignRequest, genesisValidatorsRoot []byte) ([]byte, error) {
```

Comment:
Is this meant to be V2 if it's Electra specific and is the only one?
```suggestion
func handleAggregateAttestationAndProofElectra(ctx context.Context, validator *validator.Validate, request *validatorpb.SignRequest, genesisValidatorsRoot []byte) ([]byte, error) {
```

Reply:
or just be, as both seems odd
```suggestion
func handleAggregateAttestationAndProofV2(ctx context.Context, validator *validator.Validate, request *validatorpb.SignRequest, genesisValidatorsRoot []byte) ([]byte, error) {
```

Reply:
hmm maybe i should just use V2